crypto/store/store_lib.c

```diff
@@ -111,9 +111,10 @@ OSSL_STORE_open_ex(const char *uri, OSSL_LIB_CTX *libctx, const char *propq,
      * elsewhere.
      */
     for (i = 0; loader_ctx == NULL && i < schemes_n; i++) {
-        OSSL_TRACE1(STORE, "Looking up scheme %s\n", schemes[i]);
+        scheme = schemes[i];
+        OSSL_TRACE1(STORE, "Looking up scheme %s\n", scheme);
```

Comment:
How was this causing a crash?

Reply:
Further down, old line 154, would crash because `i == schemes_n`, i.e. a classic fence post error.

Reply:
Ta - couldnt see the end of the for loop here...